Library/Formula/polo.rb

```diff
@@ -0,0 +1,33 @@
+require "language/go"
+
+class Polo < Formula
+  desc "Static blog generator compatible with Jekyll & Pelican content"
+  homepage "https://github.com/agonzalezro/polo"
+  url "https://github.com/agonzalezro/polo/archive/v1.0.tar.gz"
+  sha256 "efe7e798212b05b76ab63f32603bc15eedd9eadec3bfbfc95e36c64ba13f00dc"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/constabulary/gb" do
+    url "https://github.com/constabulary/gb.git",
+        :revision => "23f405f2981056a97abb52b5b7e528a6cddef1bb"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd "src/github.com/constabulary/gb" do
+      system "go", "get", "./..."
+    end
+
+    system "bin/gb", "build"
+    bin.install "bin/polo"
+  end
+
+  test do
+    system "wget", "https://github.com/agonzalezro/polo/archive/v1.0.tar.gz"
+    system "tar", "zxf", "v1.0.tar.gz"
+    system "cd", "polo-1.0/example", ";", "#{bin}/polo", ".", "/tmp/polo-test"
```

Comment:
Yeah I was looking for a way to download the example alone but we can’t without re-downloading the archive; Tried with the `config.json` file only and it worked :smile: